### PR TITLE
cypress test done with custom function

### PR DESCRIPTION
**Pull request by AlexMelov**

### CODE REVIEW

File: frontend/cypress.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"integrationFolder": "cypress/integration",
-	"supportFile": "cypress/support/index.ts",
+	"supportFile": "cypress/support/commands.ts",
```
Comment (henryruhs) [inline frontend/cypress.json]: This is not the way described in the docs. Keep the index.ts file and import command.ts in that file.

File: frontend/cypress/support/commands.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { environment } from '../../src/environments/environment';
+
+Cypress.Commands.add('login', () =>
+{
+	cy.visit(environment.baseUrl + '/' + environment.pageRoutes.login);
+	cy.get('[data-test="login_email"]').type('test@test.com');
+	cy.get('[data-test="login_password"]').type('123456789');
+	cy.get('[data-test="login_button"]').click();
+	cy.wait(2000);
+	cy.reload();
```
Comment (henryruhs) [inline frontend/cypress/support/commands.ts]: never use reload, this is not needed in a angular app

File: frontend/cypress/support/index.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+declare namespace Cypress
+{
+	interface Chainable<Subject>
+	{
+		login() : Chainable<Subject>
+	}
+}
+Cypress.LocalStorage.clear();
```
Comment (henryruhs) [inline frontend/cypress/support/index.ts]: this should be inside a command.types.ts

```
namespace Cypress
{
	interface Chainable
	{
		login: void;
	}
}
```

Comment (henryruhs) [inline frontend/cypress/support/index.ts]: clearing the localstorage should be not done here - you could break features with that - introduce a logout command instead if you need this

File: frontend/src/app/authentication/login/login.component.html [hunk 4]
```diff
@@ -2,14 +2,14 @@
 <form [formGroup]="form" class="form-login" (submit)="sendLogin()">
 	<mat-form-field appearance="fill">
 		<mat-label for="email">{{ 'authentication.email' | transloco }}</mat-label>
-		<input type="email" formControlName="email" id="email" matInput required email/>
+		<input type="email" formControlName="email" id="email" matInput required email data-test="login_email"/>
```
Comment (henryruhs) [inline frontend/src/app/authentication/login/login.component.html]: login-email and login-password instead of underscore

**Review by henryruhs — CHANGES_REQUESTED** [review]

File: frontend/cypress/support/index.ts [hunk 5]
```diff
@@ -1,8 +1,11 @@
...
 declare namespace Cypress
 {
 	interface Chainable<Subject>
 	{
 		login() : Chainable<Subject>
+		logout() : Chainable<Subject>
 	}
 }
-Cypress.LocalStorage.clear();
+Cypress;
```
Comment (henryruhs) [inline frontend/cypress/support/index.ts]: What is Cypress; at the end doing?

Comment (AlexMelov) [reply frontend/cypress/support/index.ts]: This is old I only have import './command';
in my index.ts

**Pull request merged**